data_util.py

```diff
@@ -33,26 +33,29 @@ def get_full_clips(data_dir, hist_len, num_clips, num_rec_out=1):
     """
     Loads a batch of random clips from the unprocessed train or test data.
     """
-    clips = np.empty([num_clips,
-                      FULL_HEIGHT,
-                      FULL_WIDTH,
-                      (3 * (hist_len + num_rec_out))])
+    clips = np.empty(
+        [num_clips, FULL_HEIGHT, FULL_WIDTH, (3 * (hist_len + num_rec_out))]
+    )
 
     # get num_clips random episodes
-    ep_dirs = np.random.choice(glob(os.path.join(data_dir, '*')), num_clips)
+    ep_dirs = np.random.choice(glob(os.path.join(data_dir, "*")), num_clips)
```

Comment:
I thought single quotes was more "pythonic". Don't really care which way we do it as long as we're consistent.